#!/usr/bin/python3
"""Task 6"""


def raise_exception_msg(message=""):
    """Raise a NameError exception with a message."""
    raise NameError(message)
